### PR TITLE
I really hope this works :)

### PR DESCRIPTION
**Pull request by veyselindi**

### CODE REVIEW

File: week4:JSweek2/2.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var Imawesome = 'I`m awesome';
+
+console.log(Imawesome);
+
+/* 
+console.log('I'm awesome');
+
+this code did not have any value above,
+and using ' caused problems. wrong format
```
Comment (nudge) [inline week4:JSweek2/2.js:7]: Do you remember using backticks ``?
For example you could use:
````
console.log(`...`);
````

File: week4:JSweek2/1.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var Eng = 'Hello World';
+let Tur = 'Merhaba Dunya';
+
+console.log(Eng);
+console.log(Tur);
+
+/* Console Log excercise */
```
Comment (nudge) [inline week4:JSweek2/1.js:7]: :+1:

File: week4:JSweek2/3.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+var x
+console.log("the value of my variable x will be: five");
```
Comment (nudge) [inline week4:JSweek2/3.js]: `five` or `5`?

File: week4:JSweek2/3.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+var x = (5);
```
Comment (nudge) [inline week4:JSweek2/3.js:3]: you don't need the parentheses, `var x = 5;` also works

File: week4:JSweek2/4.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+let y = 'isJSreallyHard?';
+console.log("the value of my string will be: isJSreallyHard?");
+console.log(y);
+y='ItLooksLikeThat';
+console.log("the value of my string will be: ItLooksLikeThat");
+console.log(y);
```
Comment (nudge) [inline week4:JSweek2/4.js:6]: :+1:

File: week4:JSweek2/5.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+var z = '7.25';
```
Comment (nudge) [inline week4:JSweek2/5.js]: this should be initialised as a number, not a string ;)

File: week4:JSweek2/5.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+console.log(z);
+var a = Math.ceil(7.25);
```
Comment (nudge) [inline week4:JSweek2/5.js]: You could use the variable you defined above, eg `Math.ceil(z)`.
Also, did you want to use `ceil` or `round` here?

File: week4:JSweek2/5.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+console.log(a);
+console.log(Math.max(z, a ));
```
Comment (nudge) [inline week4:JSweek2/5.js]: nit: you have an extra space in the line, you can remove it to make the code neater!

File: week4:JSweek2/6.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+let arr = [ ];
```
Comment (nudge) [inline week4:JSweek2/6.js]: a better name might be `myFavouriteAnimals`, it is more descriptive

File: week4:JSweek2/6.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+console.log("the value of my array will be: undefined");
+console.log(arr[0]);
+arr = ['parrot', 'fish', 'dog'];
+console.log(arr[2]);
+arr.push('baby pig');
+console.log(arr[3]);
```
Comment (nudge) [inline week4:JSweek2/6.js]: You can also try log the entire array itself, eg `console.log(arr)`. What do you expect it to show?

File: week4:JSweek2/7.js [hunk 11]
```diff
@@ -0,0 +1,4 @@
+let myString = "this is a test";
+console.log(myString);
+'myString'.length;
```
Comment (nudge) [inline week4:JSweek2/7.js]: ^ you should delete this line

**Comment by nudge** [comment]
Nice work! Added a few comments. If you can address them that would be great!

**Comment by nudge** [comment]
Also for next time, remember to create a branch from `master` for your homework. This makes it easier if you want to work on multiple week's homework at the same time without your previous week's homework getting in the way.

File: week4:JSweek2/2.js (hunk 1 above)
Comment (veyselindi) [reply week4:JSweek2/2.js:7]: I see what I did wrong.

File: week4:JSweek2/3.js (hunk 3 above)
Comment (veyselindi) [reply week4:JSweek2/3.js]: the homework says "explains in words". I misinterpreted :)

File: week4:JSweek2/3.js (hunk 4 above)
Comment (veyselindi) [reply week4:JSweek2/3.js:3]: sure. I deleted the first line of the code and it still works. I do not necessarily need to declare x like that right?
in other words, is    var x = 5    enough?

File: week4:JSweek2/5.js (hunk 6 above)
Comment (veyselindi) [reply week4:JSweek2/5.js]: got rid of ' ' :)

File: week4:JSweek2/5.js (hunk 7 above)
Comment (veyselindi) [reply week4:JSweek2/5.js]: true, it does not necessarily ask me to round up. just round. need to simplify my thinking process :)

File: week4:JSweek2/6.js (hunk 10 above)
Comment (veyselindi) [reply week4:JSweek2/6.js]: I dont even know why I referred to specific items.  :))))

**Pull request closed without merging**